pages/events.jsx

```diff
@@ -26,67 +26,69 @@ var EventsPage = React.createClass({
+              </div>
             </div>
-          </div>
-          <div className="row">
-            <div className="col-sm-offset-2 col-sm-8 col-md-offset-2 col-md-8 col-lg-offset-2 col-lg-8">
-              <p className="callout-heading">Check out the highlights from Maker Party and see more photos in our <a href="https://www.flickr.com/photos/mozilladrumbeat/galleries/72157643962655534/">Flickr gallery</a>.</p>
+            <div className="row">
+              <div className="col-sm-offset-2 col-sm-8 col-md-offset-2 col-md-8 col-lg-offset-2 col-lg-8">
+                <p className="callout-heading">Check out the highlights from Maker Party and see more photos in our <a href="https://www.flickr.com/photos/mozilladrumbeat/galleries/72157643962655534/">Flickr gallery</a>.</p>
+              </div>
             </div>
-          </div>
-          <section>
-            <IconLinks>
-              <IconLink
-                linkTo="event-resources"
-                imgSrc="/img/events-page/icon-curriculum.svg"
-                imgAlt="icon resources"
-                head="Event Resources"
-                subhead="Plan a unique event"
-              />
-              <IconLink
-                href="http://discourse.webmaker.org/category/maker-party"
-                imgSrc="/img/events-page/icon-connect.svg"
-                imgAlt="icon connect"
-                head="Join the Conversation"
-                subhead="Talk to others about your event"
-              />
-              <IconLink
-                href="https://events.webmaker.org/"
-                imgSrc="/img/events-page/icon-add-event.svg"
-                imgAlt="icon add event"
-                head="Add Your Event"
-                subhead="Join the global movement"
-              />
-            </IconLinks>
+            <section>
+              <IconLinks>
+                <IconLink
+                  linkTo="event-resources"
+                  imgSrc="/img/events-page/icon-curriculum.svg"
+                  imgAlt="icon resources"
+                  head="Event Resources"
+                  subhead="Plan a unique event"
+                />
+                <IconLink
+                  href="http://discourse.webmaker.org/category/maker-party"
+                  imgSrc="/img/events-page/icon-connect.svg"
+                  imgAlt="icon connect"
+                  head="Join the Conversation"
+                  subhead="Talk to others about your event"
+                />
+                <IconLink
+                  href="https://events.webmaker.org/"
+                  imgSrc="/img/events-page/icon-add-event.svg"
+                  imgAlt="icon add event"
+                  head="Add Your Event"
+                  subhead="Join the global movement"
+                />
+              </IconLinks>
+            </section>
           </section>
-        </section>
+        </div>
```

Comment:
I basically just wrapped https://github.com/mmmavis/teach.webmaker.org/blob/d0011c069880cf9c7053499fb1f5935db2bb03d8/pages/events.jsx#L30-L90 in `<div className="inner-container"></div>`